source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/franka.py

```diff
@@ -81,6 +81,8 @@
 FRANKA_PANDA_HIGH_PD_CFG.actuators["panda_shoulder"].damping = 80.0
 FRANKA_PANDA_HIGH_PD_CFG.actuators["panda_forearm"].stiffness = 400.0
 FRANKA_PANDA_HIGH_PD_CFG.actuators["panda_forearm"].damping = 80.0
+FRANKA_PANDA_HIGH_PD_CFG.actuators["panda_hand"].stiffness = 200.0  # for soft objects picking
+FRANKA_PANDA_HIGH_PD_CFG.actuators["panda_hand"].damping = 25.0  # for soft objects picking
```

Comment:
Hi, 

the HIGH_PD franka should be fine for training since current tasks "Isaac-Lift-Cube-Franka-v0" and "Isaac-Lift-Cube-Franka-Play-v0" still require the normal FRANKA_PANDA_CFG.

But it is problematic to train with the soft objects in the scene. I tried to set `replicate_physics=False` as done in [here](https://github.com/isaac-sim/IsaacLab/commit/361578f2960d1f42a914ea5824d4e8a9be796ce9#diff-021e2e3f962c5d996859058bad4ab41be86c7f51c73d2f32755a6f59364346d7). But it is still problematic in training:

```
2024-10-08 08:49:07 [20,847ms] [Error] [omni.physx.plugin] PhysX error: PxScene::addActor(): Soft body exceeds maximum number of softbodies per scene (PX_MAX_NB_SOFTBODY)!, FILE /builds/omniverse/physics/physx/source/physx/src/NpScene.cpp, LINE 1492
2024-10-08 08:49:16 [29,962ms] [Error] [omni.physx.plugin] PhysX error: PxSoftBody::getGpuSoftBodyIndex: Soft body must be in a scene., FILE /builds/omniverse/physics/physx/source/physx/src/NpSoftBody.cpp, LINE 102
```